src/types/parity_peers.rs

```diff
@@ -18,8 +17,34 @@ pub struct ParityPeerType {
     pub peers: Vec<ParityPeerInfo>,
 }
 
+/// add functions for ParityPeerType
```

Comment:
All the helpers methods should be available only for tests (or rather just rewrite the test above as suggested and remove the helpers)